docs/core/testing/unit-testing-platform-intro.md

```diff
@@ -183,56 +183,72 @@ To run a test project in CI add one step for each test executable that you wish
 
 - **`--diagnostic-verbosity`**
 
-Defines the verbosity level when the `--diagnostic` switch is used. The available values are `Trace`, `Debug`, `Information`, `Warning`, `Error`, or `Critical`.
+  Defines the verbosity level when the `--diagnostic` switch is used. The available values are `Trace`, `Debug`, `Information`, `Warning`, `Error`, or `Critical`.
+
+- **`--exit-on-process-exit`**
+
+  Exit the test process if dependent process exits. PID must be provided.
 
 - **`--help`**
 
-Prints out a description of how to use the command.
+  Prints out a description of how to use the command.
 
 - **`-ignore-exit-code`**
 
-Allows some non-zero exit codes to be ignored, and instead returned as `0`. For more information, see [Ignore specific exit codes](./unit-testing-platform-exit-codes.md#ignore-specific-exit-codes).
+  Allows some non-zero exit codes to be ignored, and instead returned as `0`. For more information, see [Ignore specific exit codes](./unit-testing-platform-exit-codes.md#ignore-specific-exit-codes).
 
 - **`--info`**
 
-Displays advanced information about the .NET Test Application such as:
+  Displays advanced information about the .NET Test Application such as:
 
-- The platform.
-- The environment.
-- Each registered command line provider, such as its, `name`, `version`, `description` and `options`.
-- Each registered tool, such as its, `command`, `name`, `version`, `description`, and all command line providers.
+  - The platform.
+  - The environment.
+  - Each registered command line provider, such as its, `name`, `version`, `description` and `options`.
+  - Each registered tool, such as its, `command`, `name`, `version`, `description`, and all command line providers.
 
-This feature is used to understand extensions that would be registering the same command line option or the changes in available options between multiple versions of an extension (or the platform).
+  This feature is used to understand extensions that would be registering the same command line option or the changes in available options between multiple versions of an extension (or the platform).
 
 - **`--list-tests`**
 
-List available tests. Tests will not be executed.
+  List available tests. Tests will not be executed.
 
 - **`--minimum-expected-tests`**
 
-Specifies the minimum number of tests that are expected to run. By default, at least one test is expected to run.
+  Specifies the minimum number of tests that are expected to run. By default, at least one test is expected to run.
 
 - **`--results-directory`**
 
-The directory where the test results are going to be placed. If the specified directory doesn't exist, it's created. The default is `TestResults` in the directory that contains the test application.
+  The directory where the test results are going to be placed. If the specified directory doesn't exist, it's created. The default is `TestResults` in the directory that contains the test application.
+
+- **`--timeout`**
```

Comment:
@Evangelink Same here. Do we want to document this? Or was it missing intentionally?

Reply:
Definitely a miss!